aks-node-controller/README.md

```diff
@@ -88,20 +88,20 @@ Clients need to provide CSE and Custom Data. [nodeconfigutils](pkg/nodeconfiguti
 
 1. Custom Data: Contains base64 encoded bootstrap configuration of type [aksnodeconfigv1.Configuration](pkg/gen/aksnodeconfig/v1) in json format which is placed on the node through cloud-init write directive.
 
-Format:
-```yaml
-#cloud-config
-write_files:
-- path: /opt/azure/containers/aks-node-controller-config.json
-  permissions: "0755"
-  owner: root
-  content: !!binary |
-   {{ encodedAKSNodeConfig }}`
-```
+    Format:
+    ```yaml
+    #cloud-config
+    write_files:
+    - path: /opt/azure/containers/aks-node-controller-config.json
+    permissions: "0755"
+    owner: root
+    content: !!binary |
+    {{ encodedAKSNodeConfig }}`
+    ```
```

Comment:
unnecessary spacing?

Reply:
It's to make the indent of the code snippets in readme correct.
<img width="760" height="358" alt="image" src="https://github.com/user-attachments/assets/a781e253-051f-41b6-9fb3-a5ade98068a4" />